### PR TITLE
 TELCODOCS-522 - Adding ZTP doc redirects for OCP 4.10 and 4.11

### PR DESCRIPTION
**Pull request by aireilly**
https://issues.redhat.com/browse/TELCODOCS-522

Adding redirects to support ZTP docs reorg. Do not deliver before https://github.com/openshift/openshift-docs/pull/49805

### CODE REVIEW

**Comment by aireilly** [comment]
cc @bscott-rh @pneedle-rh @gaurav-nelson PTAL

**Comment by bscott-rh** [comment]
The redirects look good to me, but we recommend creating the rules with `302` to mark temporary redirects. We migrate `302`'s to `301`'s after a month or so.

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -579,6 +579,11 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/support/summarizing-cluster-specifications.html /dedicated/osd_support/osd-summarizing-cluster-specifications.html [NE,R=301]
     RewriteRule ^dedicated/support/?(.*)$ /dedicated/osd_support/$1 [NE,R=301]
 
+    # Redirects for ZTP changes delivered in https://github.com/openshift/openshift-docs/pull/35889
+    RewriteRule ^scalability_and_performance/ztp-deploying-disconnected.html /scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.html [NE,R=302]
```
Comment (gaurav-nelson) [inline .s2i/httpd-cfg/01-commercial.conf:583]: @bscott-rh Wouldn't these require `^(container-platform)/(4\.10|4\.11)/` part as well right ? I'm using an example value for versions.

Comment (aireilly) [reply .s2i/httpd-cfg/01-commercial.conf:583]: Ah you are right I think. Will update.

Comment (aireilly) [reply .s2i/httpd-cfg/01-commercial.conf]: What about 
```
   RewriteRule ^container-platform/(4\.10|4\.11)/scalability_and_performance/ztp-deploying-disconnected.html container-platform/$1/scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.html [NE,R=302]
```

Comment (bscott-rh) [reply .s2i/httpd-cfg/01-commercial.conf]: Updated rules look good in the tester.

Comment (aireilly) [reply .s2i/httpd-cfg/01-commercial.conf]: Great. This PR will need to hang out until I get closer to merging the content update PR.

**Pull request merged**